docs/node/daemon/install.mdx

```diff
@@ -25,6 +25,190 @@ Open a terminal and run the following at the command prompt:
 kwild --version
 ```
 
-If installed correctly, it should display the version info match your downloaded version.
+If installed correctly, the displayed version info should match your downloaded version.
+
+## PostgreSQL
+
+Kwil DB is built on PostgreSQL, and requires an external PostgreSQL host running and configured for `kwild`.
+
+`kwild` currently requires PostgreSQL 16, from at least v16.1. Future major
+versions will require an update to Kwil.
+
+### PostgreSQL Installation
+
+Standard PostgreSQL installers and operating system packages are supported,
+given a supported version. We also provide Docker Compose service definitions
+with the required settings, roles, and databases pre-configured for ease of
+deployment and evaluation.
+
+The recommended options are as follows.
+
+1. System installation of `postgres`. This requires some manual configuration.
+2. The `postgres` Docker Compose service provided in the
+   `deployments/compose/postgres` folder in the [`kwil-db` source code repository](https://github.com/kwilteam/kwil-db).
+   This is pre-configured for `kwild` and is convenient for quick setup, but
+   may be less reliable or secure than a manually configured `postgres` instance.
+3. The `kwil` Docker Compose services provided in
+   `deployments/compose/kwil`. This launches both a `postgres` service
```

Comment:
Or (to keep the things we have to release less) we can add an instruction for how to build the docker image from `kwil-db` repo? Although that may be cumbersome because they have to clone and install `kwil-db` tools.

Reply:
I'm not sure what needs to be done to let it pull the `kwild:latest` image without login.

But @Yaiba and I figured out a one-liner that could work for the Quickstart page (at least the single node):

```
curl -s https://raw.githubusercontent.com/kwilteam/kwil-db/main/deployments/compose/kwil/docker-compose.yml | docker compose -f - up
```

Reply:
> (2) I was able to initialize a few containers with an old image (however the images did not start because they were missing some flags). I noticed that a postgres container did not start. Should a postgres container start if you run docker compose in the `deployments/compose/kwil` directory?

Oh, this was because https://github.com/kwilteam/kwil-db/pull/547 was not yet merged.  It's in now.